src/layout/List/ListComponent.tsx

```diff
@@ -54,16 +57,23 @@ export const ListComponent = ({ node }: IListProps) => {
 
   const { data } = useDataListQuery(filter, dataListId, secure, mapping, queryParameters);
   const bindings = item.dataModelBindings ?? ({} as IDataModelBindingsForList);
-  const { formData, setValues } = useDataModelBindings(bindings);
+
+  const { formData, setValues } = useDataModelBindings(bindings, 1, 'raw');
```

Comment:
You should probably use `DEFAULT_DEBOUNCE_TIMEOUT` instead of passing `1` here - that will preserve the default value. 🙌